### PR TITLE
RAC-6553 Port add & delete volume redfish api

### PR DESCRIPTION
**Pull request by yaolingling**
**Background**
      To port add & delete volume redfish api, the graph need to be replaced with new graphs. Details in https://rackhd.atlassian.net/browse/RAC-6553.

**Reviewers**
@nortonluo @lanchongyizu @AlaricChan

### CODE REVIEW

File: lib/api/redfish-1.0/systems.js [hunk 1]
```diff
@@ -240,9 +240,7 @@ var getSystem = controller(function(req, res) {
         if(result.vendor === 'Dell'){
             return Promise.props({
                 hardware: dataFactory(identifier, 'hardware'),
-                boot: dataFactory(identifier, 'boot'),
                 chassis: dataFactory(identifier, 'chassis'),
-                chassisData: dataFactory(identifier, 'chassisData'),
```
Comment (lanchongyizu) [inline lib/api/redfish-1.0/systems.js]: Why delete these two sources?

Comment (yaolingling) [reply lib/api/redfish-1.0/systems.js]: It was deleted carelessly.

File: lib/api/redfish-1.0/systems.js [hunk 2]
```diff
@@ -1918,19 +1916,22 @@ var addVolume = controller(function(req,res) {
...
                     case "Mirrored":
-                        graphOptions.defaults.raidLevel = 'r1';
+                        graphOptions.defaults.raidLevel = 1;
                         break;
                     case "StripedWithParity":
-                        graphOptions.defaults.raidLevel = 'r5';
+                        graphOptions.defaults.raidLevel = 5;
+                        break;
+                    case "RawDevice":
+                        graphOptions.defaults.raidLevel = 6;
```
Comment (lanchongyizu) [inline lib/api/redfish-1.0/systems.js]: RawDevice means that `The volume is a raw physical device without any RAID or other virtualization applied.`  https://github.com/RackHD/on-http/blob/master/static/DSP8010_2016.3/metadata/Volume_v1.xml#L180-L182
Please remove RawDevice here.

Comment (yaolingling) [reply lib/api/redfish-1.0/systems.js]: ok

File: spec/lib/api/redfish-1.0/system-spec.js [hunk 3]
```diff
@@ -3084,6 +3084,74 @@ describe('Redfish Systems Root', function () {
...
+                    {
+                        "@odata.id": "/redfish/v1/Systems/SomeNodeId/Storage/RAID.Integrated.1-1/Drives/1"
+                    },
+                    {
+                        "@odata.id": "/redfish/v1/Systems/SomeNodeId/Storage/RAID.Integrated.1-1/Drives/2"
+                    }]
+                }
+            }
+        };
+        waterline.catalogs.findLatestCatalogOfSource.resolves(Promise.resolve({
```
Comment (pengz1) [inline spec/lib/api/redfish-1.0/system-spec.js]: Promise.resolve() is not necessary, any value returned from promise is a promise already.

File: spec/lib/api/redfish-1.0/system-spec.js [hunk 4]
```diff
@@ -3084,6 +3084,74 @@ describe('Redfish Systems Root', function () {
...
+                "Name": "name 123",
+                "CapacityBytes": 1234567,
+                "VolumeType": "NonRedundant",
+                "Links": {
+                    "Drives@odata.count": 3,
+                    "Drives": []
+                }
+            }
+        };
+        waterline.catalogs.findLatestCatalogOfSource.resolves(Promise.resolve({
```
Comment (pengz1) [inline spec/lib/api/redfish-1.0/system-spec.js]: Promise.resolve() is not necessary, any value returned from promise is a promise already.

Comment (pengz1) [reply spec/lib/api/redfish-1.0/system-spec.js]: Generally, it is not a good idea to throw 500 for user input error, 400 is more reasonable.

File: spec/lib/api/redfish-1.0/system-spec.js (hunk 3 above)
Comment (yaolingling) [reply spec/lib/api/redfish-1.0/system-spec.js]: ok

File: spec/lib/api/redfish-1.0/system-spec.js (hunk 4 above)
Comment (yaolingling) [reply spec/lib/api/redfish-1.0/system-spec.js]: ok

**Pull request merged**